app/i18n/en/admin.php

```diff
@@ -112,6 +112,13 @@
 		),
 		'title' => 'Extensions',
 		'user' => 'User extensions',
+		'community' => 'Available community extensions for download',
```

Comment:
Either just `Available community extensions` or `Community extensions available for download`.